manifests/bosh-manifest/reference-bosh-manifest.yml

```diff
@@ -160,10 +206,14 @@ properties:
       0000000000000000000000000000000000000000000000000000000000000000
       0000000000000000000000000000000000000000000000000000000000000000
       -----END RSA PRIVATE KEY-----
+
 releases:
 - name: bosh
   sha1: 6b12652650b87810dcef1be1f6a6d23f1c0c13a7
   url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=255.8
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
```

Comment:
@alext - there is a [corresponding pull request](https://github.com/alphagov/paas-collectd-boshrelease/pull/1) in the paas-collectd-boshrelease repository for upgrading to 0.4